it/mail/chrome/messenger/messenger.properties

```diff
@@ -78,60 +78,60 @@ PrintPreviewMessage=Creazione anteprima in corso…
-deletingMsgsFailed=Impossibile eliminare i messaggi nella cartella %S in quanto è utilizzata da qualche altra operazione. Attendere che tale operazione venga completata e riprovare.
+parsingFolderFailed=Impossibile aprire la cartella %S in quanto è utilizzata da qualche altra operazione. Attendi che tale operazione sia completata, quindi riseleziona la cartella.
+deletingMsgsFailed=Impossibile eliminare i messaggi nella cartella %S in quanto è utilizzata da qualche altra operazione. Attendi che tale operazione sia completata e riprova.
 alertFilterCheckbox=Non avvisare di nuovo.
-compactFolderDeniedLock=La cartella '%S' non può essere compattata a causa di un'altra operazione in corso. Riprovare più tardi.
-compactFolderWriteFailed=La cartella '%S' non può essere compattata in quanto non è possibile scriverci. Verificare di avere spazio su disco sufficiente e avere privilegi in scrittura, quindi riprovare.
-filterFolderWriteFailed=I messaggi non possono essere filtrati nella cartella '%S' perché non è riuscita la scrittura nella cartella. Verificare di avere sufficiente spazio libero su disco, e sufficienti privilegi in scrittura. Quindi riprovare.
-copyMsgWriteFailed=Il messaggio non può essere spostato o copiato nella cartella '%S' in quanto non è stato possibile scriverci. Per aumentare lo spazio a disposizione nel disco, dal menù FIle, selezionare prima Svuota cestino, quindi Compatta cartelle. Dopodiché riprovare.
-cantMoveMsgWOBodyOffline=Mentre si lavora non in linea, non si possono spostare o copiare messaggi che non sono stati scaricati per essere usati non in linea. Dalla finestra di Posta, aprire il menu File, scegliere Non in linea, togliere il segno di spunta da "Lavora non in linea" e riprovare.
-operationFailedFolderBusy=L'operazione non è riuscita perché un'altra operazione sta usando la cartella. Attendere la fine di tale operazione e riprovare.
-folderRenameFailed=La cartella non può venir rinominata. Potrebbe essere in uso, o il nuovo nome non è valido.
+compactFolderDeniedLock=La cartella '%S' non può essere compattata a causa di un'altra operazione in corso. Riprova più tardi.
+compactFolderWriteFailed=La cartella '%S' non può essere compattata in quanto non è possibile scriverci. Verifica di avere spazio su disco sufficiente e di avere privilegi in scrittura, quindi riprova.
+filterFolderWriteFailed=I messaggi non possono essere filtrati nella cartella '%S' perché non è riuscita la scrittura nella cartella. Verifica di avere sufficiente spazio libero su disco, e sufficienti privilegi in scrittura. Quindi riprova.
+copyMsgWriteFailed=Il messaggio non può essere spostato o copiato nella cartella '%S' in quanto non è stato possibile scriverci. Per aumentare lo spazio a disposizione nel disco, dal menù File, seleziona prima Svuota cestino, quindi Compatta cartelle. Quindi riprova.
```

Comment:
CORRECTIONS NEEDED. Please modify the second part of the sentence as follows: 

"Per aumentare lo spazio a disposizione sul disco, dal menù File seleziona prima Svuota cestino, poi Compatta cartelle, infine riprova."